bin/yb-ctl

```diff
@@ -1221,7 +1251,7 @@ class ClusterControl:
         command_list += [
             # Add in all the shared flags
             "--fs_data_dirs \"{}\"".format(",".join(node_base_dirs)),
-            "--webserver_interface {}".format(self.options.get_ip_address(daemon_id)),
+            "--webserver_interface {}".format(self.options.get_client_listen_host_port(daemon_id)),
             "--rpc_bind_addresses {}".format(self.options.get_ip_address(daemon_id)),
```

Comment:
Does this need to be changed as well?

Reply:
I intentionally didn't want to change rpc_bind_address (coz then I would need to supply server_broadcast_addresses in these scenarios)